packages/postcss-minify-gradients/src/isColorStop.js

```diff
@@ -0,0 +1,52 @@
+import { unit } from 'postcss-value-parser';
+import { colord, extend } from 'colord';
+import namesPlugin from 'colord/plugins/names';
+
+extend([namesPlugin]);
+
+/* Code derived from https://github.com/pigcan/is-color-stop */
```

Comment:
Added a link to the orignal code, I don't think the MIT license requires it but I agree it is nicer to have proper attribution

Reply:
👍